build/check-images.js

```diff
@@ -119,9 +120,24 @@ function checkImageReferences(doc, $, options, { url, rawContent }) {
           // it now, we still want the full relative URL.
           img.attr("src", absoluteURL.pathname);
         } else {
+          let suggestion = null;
+          // If this document is *not* en-US, perhaps the external image has already
+          // been downloaded by the en-US equivalent. If so, make that the suggestion.
+          if (doc.locale !== DEFAULT_LOCALE) {
+            const filePath = Image.findByURL(
+              path.join(
+                doc.mdn_url.replace(`/${doc.locale}/`, `/${DEFAULT_LOCALE}/`),
+                path.basename(src)
+              )
```

Comment:
Is this really the best way to "hop over" to the en-US content? It certainly seems to work :)

Reply:
Best way for now 🤷